shared/profile/edit-profile/render.desktop.js

```diff
@@ -0,0 +1,66 @@
+/* @flow */
+import React, {Component} from 'react'
+import {BackButton, Box, Button, Input} from '../../common-adapters'
+import {globalStyles} from '../../styles/style-guide'
+import type {Props} from './render'
+
+class Render extends Component<void, Props, void> {
+  render () {
+    return (
+      <Box style={styleOuterContainer}>
+        {this.props.onBack && <BackButton onClick={this.props.onBack} style={{position: 'absolute', left: 10, top: 10, zIndex: 12}} />}
+        <Box style={styleContainer}>
+          <Input
+            autoFocus
+            style={styleEditProfile}
+            floatingLabelText='Full name' value={this.props.fullname}
+            onEnterKeyDown={this.props.onSubmit}
+            onChangeText={fullname => this.props.onFullnameChange(fullname)} />
+
+          <Input
+            style={styleEditProfile}
+            floatingLabelText='Location'
+            value={this.props.location}
+            onEnterKeyDown={this.props.onSubmit}
+            onChangeText={location => this.props.onLocationChange(location)} />
+
+          <Input
+            style={styleEditProfile}
+            floatingLabelText='Bio'
+            value={this.props.bio}
+            errorText={this.props.bioLengthLeft <= 5 ? this.props.bioLengthLeft + ' characters left.' : ''}
+            onEnterKeyDown={this.props.onSubmit}
+            onChangeText={bio => this.props.onBioChange(bio)} />
```

Comment:
This input is multiline in the design, and has a smaller font than the other inputs

<img width="356" alt="screen shot 2016-07-20 at 10 14 48 am" src="https://cloud.githubusercontent.com/assets/1152104/16995737/d468e616-4e62-11e6-8689-59ab0f856636.png">